### DISCUSSION TITLE
Add Victory moves

### DISCUSSION THREAD

**Discussion by KiII-ReaI** [spicyboys/drg-completionist#194] [Ideas]
Title. As some victory moves are obtained through OC, Packs and Creates, they should be added too.

**Comment by rob0rt** [comment]
We had those in the old version but never ported them over to the new one. My biggest dilemma was not knowing how to represent them in the UI since there's so many of them and their icons aren't particularly meaningful. But I suppose in the spirit of perfect being the enemy of done, even a basic checkbox list would suffice.

**Reply by KiII-ReaI** [reply]
Oh, thank you for reply! I did not expect to get any, as you mentioned not to work on this anymore. Yeah, a simple checkbox would suffice. Or you can draw unique icons for each victory move, but that would take to much time.

**Reply by rob0rt** [reply]
I still keep an eye on things, even if I'm not actively working on this. I make no promises for when I might address this of course but it is nice to have a list of things that can be improved.
